moto/sqs/responses.py

```diff
@@ -242,25 +242,59 @@ def send_message_batch(self):
 
         queue_name = self._get_queue_name()
 
+        try:
+            self.sqs_backend.get_queue(queue_name)
+        except QueueDoesNotExist as e:
```

Comment:
Is this needed? If it raises a `QueueDoesNotExist`, that should have all the info required to format the exception to the client.

Reply:
The problem is the general implementation of the `QueueDoesNotExist` exception. It results in an 503 error while running the moto in server mode. I would need to change it to a `RESTError` then the handling would be right, I think.

Reply:
Thank You -- that makes sense.